sdcm/keystore.py

```diff
@@ -38,5 +39,13 @@ def get_qa_ssh_keys(self):
         # TODO
         pass
 
+    def get_dbaas_lab_keys(self):
+        key_file = Path('~/.ssh/dbaas_lab').expanduser()
```

Comment:
1) class Keystore shouldn't has nothing to do with filesystem paths and ssh keys. The purpose of the class
is to retrieve information from the Secret store. So please separate getting ssh key from the other things.
2) Need to avoid using words like dbaas or similar

I suggest adding generic method `get_json_file_secret(file_name)` ( or rename `_get_json`, or make it public method)
and call it from the "plugin code" like `get_json_file_secret("dbaas_lab")` ( `get_json("dbaas_lab")`).
Secondly, download and handle the ssh key in the "plugin code" (call download_file method on KeyStore object)

Reply:
fix, and remove this code